app/controllers/comments_controller.rb

```diff
@@ -13,6 +13,20 @@ def create
     end
   end
 
+  def edit
+    @comment = Comment.find(params[:id])
+  end
+
+  def update
+    @comment = Comment.find(params[:id])
```

Comment:
Same as above, scope this to the current user to prevent unauthorized edits being made.